### ISSUE TITLE
libmemcached and libevent packages

### ISSUE THREAD

**Issue by dexoid** [essentialkaos/kaos-repo#191]
Hello.
With package nfs-utils was installed libevent-2.1.8-2.
But I can not install opendkim package:
```
--> Running transaction check
---> Package opendkim.x86_64 0:2.11.0-0.1.el7 will be installed
--> Processing Dependency: libmemcachedutil.so.2()(64bit) for package: opendkim-2.11.0-0.1.el7.x86_64
--> Processing Dependency: libmemcached.so.11()(64bit) for package: opendkim-2.11.0-0.1.el7.x86_64
--> Running transaction check
---> Package libmemcached.x86_64 0:1.0.16-5.el7 will be installed
--> Processing Dependency: libevent-2.0.so.5()(64bit) for package: libmemcached-1.0.16-5.el7.x86_64
--> Finished Dependency Resolution
Error: Package: libmemcached-1.0.16-5.el7.x86_64 (base)
           Requires: libevent-2.0.so.5()(64bit)
           Available: libevent-2.0.22-2.el7.x86_64 (kaos-release)
               libevent-2.0.so.5()(64bit)
           Installed: libevent-2.1.8-2.el7.x86_64 (@kaos-release)
              ~libevent-2.1.so.6()(64bit)
```

**Comment by andyone** [comment]
Hi! There is a problem with `libmemcached`, not with `opendkim` itself. `libmemcached` linked with an old version of `libevent` (2.1.8 was released 1.5 years ago).

The main idea of our repository — have the latest version of packages with all actual fixes and improvements. We can't keep compatibility with all outdated packages from the base repository.

I'll try to find spec file for `libmemcached` and add it to our repository but I can't give any ETA at this moment.

**Comment by andyone** [comment]
Fixed by #196
